test/runtests.jl

```diff
@@ -195,6 +194,18 @@ end
         58201.07894736842,
     ]
     y_binned = bucket(X, y, bins; reduction = mean)
+    @test y_binned = [0.00019574174576036845, 0, 0, 0, 0]
```

Comment:
This has a floating point comparison error on the CI: try using the `\approx` equals and adding an `atol` or `rtol` parameter to the macro:

```julia
@test y_binned ≈ [0.00019574174576036845, 0, 0, 0, 0] rtol = 1e-3
```

Reply:
I've gone with `atol = 1e-3` here and below. That should be fine as the error when the code wasn't working was much larger.